cmake/BLTMacros.cmake

```diff
@@ -664,65 +664,51 @@ macro(blt_append_custom_compiler_flag)
 endmacro(blt_append_custom_compiler_flag)
 
 
-
-
 ##------------------------------------------------------------------------------
-## blt_find_libraries( FOUND_LIBS <FOUND_LIBS>
-##                     REQUIRED_NAMES [libname1 [libname2 ...]]
-##                     OPTIONAL_NAMES [libname1 [libname2 ...]]
+## blt_find_libraries( FOUND_LIBS <FOUND_LIBS variable name>
+##                     NAMES [libname1 [libname2 ...]]
+##                     OPTIONAL [TRUE | FALSE ]
```

Comment:
Can we make the keyword be ``REQUIRED`` instead of ``OPTIONAL``, with a default of ``TRUE`` ?
This seems like a better fit with cmake's ``find_package`` -- https://cmake.org/cmake/help/v3.8/command/find_package.html -- although, in that case, it is an option, and the default is OFF.